model/stream.go

```diff
@@ -30,6 +30,7 @@ type Stream struct {
 	CourseID         uint
 	Start            time.Time `gorm:"not null"`
 	End              time.Time `gorm:"not null"`
+	ChatEnabled      bool      `gorm:"not null;default:false"`
```

Comment:
This should be actually nullable, otherwise we always default to false (even when the is enabled globally), which might not be what we want when creating a new stream/lecture.
```suggestion
	ChatEnabled      bool      `gorm:"default:null"`
```